spec/support/helpers.rb

```diff
@@ -24,6 +24,7 @@ def stub_omniauth(disable_completely: true, user: nil)
     # Temp solution until we implement `return_url` for DfE sign-in
     if disable_completely
       allow_any_instance_of(ApplicationController).to receive(:authenticate)
+      Thread.current[:manage_courses_backend_token] = 'foobar'
```

Comment:
Temporary change to get the specs green. But this will be removed in the next PR that removes this method completely